Formula/liquid-dsp.rb

```diff
@@ -0,0 +1,54 @@
+class LiquidDsp < Formula
+  desc "Signal processing (DSP) library for software-defined radios"
+  homepage "http://liquidsdr.org/"
+  url "https://github.com/jgaeddert/liquid-dsp/archive/v1.2.0.tar.gz"
+  sha256 "783854b63f5e9a9830dcd95b2c1d5c0bef7d10d22a0e27844a354b59283b6de3"
+
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "fftw" => :recommended
+  # libfec could be an optional dependency, but it's not available, yet
+
+  fails_with :clang do
```

Comment:
If it will apply cleanly, we'd prefer the:

```ruby
patch do
  url "https://github.com/jgaeddert/liquid-dsp/pull/51.patch"
  sha256 ""
end
```
Format.